File: leetcode/two-sum.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {*[]}
 */
const twoSum = function (nums, target) {
    let len = nums.length
    let a = [];
    for (let i = 0; i < len; i++) {
        for (let j = i; j < len; j++) {
            if (nums[i] + nums[j] == target) {
                a[0] = i;
                a[1] = j;
                return a;
                // break;
            }
        }
    }
};

let c=twoSum([1,3,4,5,6],7);
console.log(c);

File: leetcode/two-sum.cpp
#include <bits/stdc++.h>
using namespace std;

int main()
{
	int nums[100],target,len=5;
	cin>>nums>>target;
    //len=nums.length();
    //len = sizeof(nums) / sizeof(nums[0]);

    for(int i=0; i<len; i++)
    {
        for(int j=i; j<len; j++)
        {
            if(nums[i]+nums[j]==target)
            {
                cout<<"["<<nums[i]<<","<<nums[j]<<"]";
                break;
            }
        }
    }
}
